app/views/studies/_shared_sync_functions.js.erb

```diff
@@ -95,6 +167,31 @@ $('#study-file-<%= study_file.id %>').on('change', '.file-type', function() {
             submitBtn.attr('disabled', false);
         }
     }
+
+    // FileUploadControl updates this more elegantly via React,
+    // but this direct approach is easier and as effective here.
+    let useConvention = false
+    if (fileType === 'Metadata') {
+      const conventionSelector =
+        '#study-file-<%= study_file.id %> ' +
+        '#study_file_use_metadata_convention_true'
+      useConvention = ($(conventionSelector)[0].checked === true)
```

Comment:
I *think* you can simplify the above code (and avoid jquery) as
`const conventionSelector = '#study-file-<%= study_file.id %> #study_file_use_metadata_convention_true'
`const useConvention = !! document.querySelector(conventionSelector)?.checked`

Reply:
This seems like an eye-of-the-beholder scenario -- to me, the existing approach looks simpler!

Reply:
but *clearly* the suggested approach is *much* better.  jkjkjk, yup, either is fine